uring-common/src/ext/impl_iobuf_for_array.rs

```diff
@@ -0,0 +1,13 @@
+unsafe impl<const N: usize> crate::buf::IoBuf for [u8; N] {
+    fn stable_ptr(&self) -> *const u8 {
+        self.as_ptr()
+    }
```

Comment:
How does one provide a stable pointer to an array (on-stack sized value)?

Maybe a `std::pin::Pin<[u8; N]>` or similar, but that'd be subject to cancellation if it lived in the future which initiates IO.

Reply:
right, yuck

Reply:
https://github.com/neondatabase/tokio-epoll-uring/pull/43/commits/61c258a40ba99e561bd8528fede922df5dacd0be